plasticos_web_lead_ai_triage/models/web_lead_triage.py

```
@@ -0,0 +1,689 @@
+                    url[:80], self.lead_id, exc,
+                )
+
+    # ═════════════════════════════════════════════════════════
+    # Manual Actions
+    # ═════════════════════════════════════════════════════════
+
+    def action_retry_triage(self):
+        """Re-run the triage pipeline on an errored or cold lead."""
+        for rec in self:
+            if rec.state not in ("error", "skipped", "received"):
+                raise UserError("Only errored, skipped, or received leads can be re-triaged.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)